utils/jwt.ts

```diff
@@ -0,0 +1,20 @@
+import getConfig from "next/config";
+import { verify } from "jsonwebtoken"
+
+export type AuthResult = {
+    user_id: string
+}
+
+const { serverRuntimeConfig } = getConfig();
+
+const isAuthValid = (token: string): AuthResult | null => {
+    try {
+        const decoded = verify(token, serverRuntimeConfig.JWT_SECRET)
+        console.log('decoded', decoded);
+        return decoded as AuthResult
+    } catch (e) {
+        return null
+    }
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAuthValid` function is currently logging the decoded JWT token to the console. This could potentially expose sensitive user information in logs, which is a security concern. It's recommended to remove this log statement.
```diff
    try {
        const decoded = verify(token, serverRuntimeConfig.JWT_SECRET)
-13:         console.log('decoded', decoded);
+13:         // Removed logging of decoded token for security reasons
        return decoded as AuthResult
    } catch (e) {
        return null
    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->